src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationCreationRequestValidator.java

```diff
@@ -16,4 +17,13 @@ public void validate(OrganisationCreationRequest organisationCreationRequest) {
         validators.forEach(v -> v.validate(organisationCreationRequest));
     }
 
+    public static boolean contains(String status) {
+        for (OrganisationStatus type : OrganisationStatus.values()) {
+            if (type.name().equalsIgnoreCase(status)) {
```

Comment:
Good. This is how we should always be comparing enums. Therefore, you need to remove the values from the OrganisationStatus enum and just use .name() instead.

Reply:
Removed from enum and used name() method every where.